plugins/woocommerce/client/admin/client/settings-payments/components/status-badge/status-badge.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { Pill } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import './status-badge.scss';
+
+interface StatusBadgeProps {
+	status:
+		| 'active'
+		| 'inactive'
+		| 'needs_setup'
+		| 'test_mode'
+		| 'recommended'
+		| 'has_incentive';
+}
+
+export const StatusBadge: React.FC< StatusBadgeProps > = ( { status } ) => {
```

Comment:
While React.FC isn't inherently harmful, it seems that the current best practice is to avoid it in favor of more explicit and flexible type definitions. Since React 18, it is generally discouraged by many developers and the React team itself. https://www.mydatahack.com/moving-away-from-fc-and-vfc/

We don't have official discussions about this topic, but we're using less and less `FC` and `VFC` in our codebase. 😅

Reply:
TIL, thanks for sharing!

Reply:
Thank you, TIL! I've never liked the `React.FC` syntax, but also wasn't sure on the most appropriate alternative. 

I've updated in fb54f9c74b - let me know if that looks good.